src/components/pages/WelcomePage.tsx

```diff
@@ -833,7 +833,11 @@ class WelcomePage extends React.Component<Props, State> {
       const start = moment(formValues["start-date-time"] as number);
       const end = moment(formValues["end-date-time"] as number);
 
+      if (this.state.defaultImageUrl || this.state.imageUrl) {
       formValues["image-url"] = this.state.imageUrl ? this.state.imageUrl : this.state.defaultImageUrl;
+      } else {
+        return alert("Lisää tapahtumalle kuva, mikäli sinulla ei ole omaa kuvaa, klikkaa kohtaa \"Näytä oletuskuvat\"");
```

Comment:
Is there a localization in this project? In case there is this string could be moved there.